spec.md

```diff
@@ -147,12 +147,24 @@ that contains both context and data).
   * MUST be a non-empty string
 
 ### source
-* Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
+* Type: String
+* Description: This property identifies the source of the event as defined
+  by the publishing organization. The value SHOULD be a URI. The source 
+  property value MUST uniquely identify the source within the scope set by the
+  "namespace" property. While the value MAY be a interpreted as a URL, the 
+  purpose of it to logically identify the source and not to establish the 
```

Comment:
"**is** to logically..." ?

Reply:
@clemensv ^^

Reply:
I don't think it makes any sense to modify source by itself.  As documented, it is an "object" that has properties `type` and `id` (listed in the spec as `source-type` and `source-id`).  I agree that, as written, the spec is confusing, but I believe we need to address how to specify the source holistically.

Reply:
I've read the table you put into the associated issue, and I can't say that it makes sense to me. The application receiving an event needs to know who sent it. A URI appears to be sufficient for that purpose. A composite identifier that has an explicit notion of an "id" and "type" is a complication.

Reply:
I agree, i dont see why we need a hierarchy (namespace, id, type), a single string value should do for source (identifying the event reporting entity/resource, not the modified S3 object names..). event specific source data should be written into the event structure itself.